com.unity.render-pipelines.high-definition/Documentation~/HDRP-Sample-Content.md

```diff
@@ -18,14 +18,6 @@ Additional Post-Processing Data gives you access to Textures you can use with po
 - Spectral Look-up Textures (designed for use in [Chromatic Aberrations](Post-Processing-Chromatic-Aberration.html)).
 - Look-Up Textures.
 
-## Shader Graph Samples
-
-This Sample includes example Shader Graphs that show you how to use the following Master Nodes: 
-
-* [Fabric Master Node](Master-Node-Fabric.html).
-* [Hair Master Node](Master-Node-Hair.html).
-* [Decal Master Node](Master-Node-Decal.html).
-
 The Fabric and Hair Master Nodes usually require various work from artists inside the Shader Graph and the Samples are a good head start.
```

Comment:
This bit is from the Shader Graph Samples section. If you're removing that section content from there, I guess you want to remove this too?

Reply:
moved it down under the material samples section